packages/commuter-client/src/index.js

```diff
@@ -12,11 +13,15 @@ import Notebook from "./Notebook";
 import Commuter from "./Commuter";
 import Discovery from "./Discovery";
 
+const history = useBasename(createHistory)({
+  basename: '/view'
```

Comment:
This makes it so discovery is at /view/discovery now which I don't think is what we want.

However this is the right API for react router v3 for handling the base path as would be used with jupyterhub's /services/commuter endpoint.